setup.py

```diff
@@ -5,6 +5,11 @@
 with open("requirements.txt") as fh:
     install_requires = fh.readlines()
 
+with open("test-requirements.txt") as fh:
+    test_install_requires = fh.readlines()
+    test_install_requires.remove("-r requirements.txt")
```

Comment:
What's this doing?

Reply:
add comment

Reply:
Its putting the packages in the "test-requirements.txt" into a list, but then removes "-r requirements.txt". However, the packages in requirements.txt is added in the next line `test_install_requires.extend(install_requires)`. All this is because pypi doesn't know how to read a given file to retrieve packages.